Replicator/ReplicatorOptions.hh

```diff
@@ -248,39 +249,47 @@ namespace litecore { namespace repl {
         inline void verify() const;
 
         size_t collectionCount() const {
-            return collectionOpts.size();
+            return _workingCollections.size();
         }
 
         Mode push(CollectionIndex i) const {
-            return collectionOpts[i].push;
+            return _workingCollections[i].push;
         }
 
         Mode pull(CollectionIndex i) const {
-            return collectionOpts[i].pull;
+            return _workingCollections[i].pull;
         }
 
         Validator pushFilter(CollectionIndex i) const {
-            return collectionOpts[i].pushFilter;
+            return _workingCollections[i].pushFilter;
         }
 
-        Validator pullFilter(CollectionIndex i) const  {
-            return collectionOpts[i].pullFilter;
+        Validator pullFilter(CollectionIndex i) const {
+            return _workingCollections[i].pullFilter;
         }
 
         void* collectionCallbackContext(CollectionIndex i) const {
```

Comment:
This appears to be the same as as callbackContextAt().

Reply:
Oh I didn't notice it because so many places were going through collectionOpts directly!

Reply:
Perhaps it's time to make collectionOpts private, _collectionOpts. It also helps find out mistaken use of it in this file.
The exiting style is, all public properties are direct members.